src/NServiceBus.RavenDB/Internal/DocumentStoreInitializer.cs

```diff
@@ -38,7 +38,9 @@ internal IDocumentStore Init(ReadOnlySettings settings)
 
         void ApplyConventions(ReadOnlySettings settings)
         {
-            var idConventions = new DocumentIdConventions(docStore, settings.GetAvailableTypes(), settings.EndpointName());
+            var sagasEnabled = settings.GetOrDefault<bool>(typeof(Features.Sagas).FullName);
```

Comment:
@DavidBoike I know this works, but to me it feels like a hack that will bit us later on, IMO I would try to use a `Feature` with `Depends`.